src/main/java/lotto/domain/LottoMoney.java

```diff
@@ -0,0 +1,82 @@
+package lotto.domain;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Objects;
+
+import lotto.exception.LottoErrorCode;
+import lotto.exception.LottoException;
+
+public class LottoMoney {
+    private static final int LOTTO_PRICE = 1000;
+    private static final int ZERO_SIZE = 0;
+    private static final int MIN_MONEY = 0;
+
+    private long money;
+
+    public LottoMoney(long money) {
+        this.money = money;
+    }
+
+    public LottoMoney(String money) {
+        this.money = parseToValidMoney(money);
+    }
+
+    private long parseToValidMoney(String moneyText) {
```

Comment:
```suggestion
    private long parseToValidMoney(String money) {
```

문자열 타입이니 text 표시는 필요 없어보이네요.